### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by faridmurshed**

### CODE REVIEW

**Comment by faridulhaque** [comment]
hello

**Comment by faridulhaque** [comment]
hi

File: README.md [hunk 1]
```diff
@@ -8,4 +8,3 @@
 * React
 * TypeScript
 * MUI
```
Comment (faridulhaque) [inline README.md:10]: fffffffffff

Comment (faridulhaque) [inline README.md:10]: pull

File: README.md [hunk 2]
```diff
@@ -8,4 +8,3 @@
...
-f
```
Comment (faridulhaque) [inline README.md:11]: ccc

Comment (faridulhaque) [inline README.md:11]: fff

File: README.md (hunk 1 above)
Comment (faridulhaque) [inline README.md:10]: hu

Comment (faridmurshed) [inline README.md:10]: very bad

Comment (faridmurshed) [reply README.md:10]: fffs

Comment (faridulhaque) [reply README.md:10]: fffd

Comment (faridmurshed) [reply README.md:10]: dddddddddd

File: README.md [hunk 3]
```diff
@@ -8,4 +8,3 @@
 * React
```
Comment (Govindaroraaa) [inline README.md:8]: Heyyy

**Pull request merged**